src/configcontainer.cpp

```diff
@@ -46,6 +46,7 @@ ConfigContainer::ConfigContainer()
 	{"cleanup-on-quit", ConfigData("yes", ConfigDataType::BOOL)},
 	{"confirm-delete-all-articles", ConfigData("yes", ConfigDataType::BOOL)},
 	{"confirm-mark-all-feeds-read", ConfigData("yes", ConfigDataType::BOOL)},
+	{"confirm-mark-feed-read", ConfigData("no", ConfigDataType::BOOL)},
```

Comment:
Please document this setting in _doc/configcommands.dsv_ (note that the file is sorted alphabetically).

Also, please change the default to `yes`. Good on you for caring about backwards compatibility, but I think the benefits of confirmation outweigh the costs for the existing users.

Reply:
Fixed!